lib/core/proxy.c

```diff
@@ -42,6 +42,8 @@ struct rp_generator_t {
     int is_websocket_handshake;
     int had_body_error; /* set if an error happened while fetching the body so that we can propagate the error */
     h2o_timer_t send_headers_timeout;
+    unsigned req_done : 1;
+    unsigned res_done : 1;
```

Comment:
I believe these need to be explicitly initialized in `proxy_send_prepare`

Reply:
fixed in https://github.com/h2o/h2o/pull/2010/commits/1ba340144364024f7e62b4907679fb4a3272c7a4